packages/react-sdk/src/components/GuidedTour/StyledJoyrideStep.tsx

```diff
@@ -88,7 +88,7 @@ export function StyledJoyrideStep({
 
       <DialogActions>
         <Typography variant="body1" color="text.secondary" flex={1}>
-          {t('guidedTour.slideCount', '{{index}} of {{total}}', {
+          {t('guidedTour.slideCount', '', {
```

Comment:
won't fix

Reply:
> I removed it in a first iteration, but then considered what would be fasted for me to do and the easiest for `git revert` in 2 weeks from now.

we will not need to revert. this will stay on the `release/mc-version` branch and never be merged into `main`